### PR TITLE
Neovide.Neovide version 0.10.3

### LINKED ISSUE #1374: Package for `winget`
It would be nice to be able to install the latest stable release of `neovide` via[`winget`](https://github.com/microsoft/winget-cli). As `winget` is developed by Microsoft, I expect with time it will become the de facto package manager for Windows.

In a soon to be released version (microsoft/winget-cli#182), `winget` will be able to install a standalone executable and add it to the `PATH`. However, [I don't believe this will support the start/search menu](https://github.com/microsoft/winget-cli/issues/182#issuecomment-1168246167). This would require `neovide` to be published as an `.exe` in releases. In future, it should be possible to extract the `.exe` from a `.zip` (microsoft/winget-cli#140), however I don't believe this will be available for a while yet.

To achieve this at the moment, we would have to bundle `neovide` into an installer such as an `.msi` which would be able to add `neovide` to the `PATH` add create a start menu shortcut. I think the easiest way to do this would be to use [`cargo-wix`](https://github.com/volks73/cargo-wix) to package `neovide` in the CI workflow.

What are the everyone's thoughts? It could be quite a while before `winget` implement all the features required, I would be happy to investigate using `cargo-wix` as a solution.

### PR DESCRIPTION
**Pull request by afermar**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
Resolves neovide/neovide#1374

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/96805)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hi @afermar, 

This package installs normally, but requires the dependency `Microsoft.VCRedist.2015+.x64` to run:
image

**Comment by fredizzimo** [comment]
@afermar
Note that we switched to a non-zipper installer in the latest 0.11.0 released. 

For the runtime it loks like we have to add some options https://volks73.github.io/cargo-wix/cargo_wix/index.html#c-runtime

**Comment by fredizzimo** [comment]
Hi. I noticed that the pipeline started for this. 

The version here is quite old though, our latest version is https://github.com/neovide/neovide/releases/tag/0.11.2

**Comment by stephengillie** [comment]
> Hi @afermar,
> 
> This package installs normally, but requires the dependency `Microsoft.VCRedist.2015+.x64` to run: image

This is still the case.

File: manifests/n/Neovide/Neovide/0.10.3/Neovide.Neovide.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Neovide.Neovide
+PackageVersion: 0.10.3
```
Comment (stephengillie) [inline manifests/n/Neovide/Neovide/0.10.3/Neovide.Neovide.installer.yaml:5]:
```suggestion
PackageVersion: 0.10.3
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment (stephengillie) [inline manifests/n/Neovide/Neovide/0.10.3/Neovide.Neovide.installer.yaml:5]:
```suggestion
PackageVersion: 0.10.3
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

**Pull request closed without merging**